zoom_connector.py

```diff
@@ -312,153 +319,176 @@ def _get_password(self, password, pass_gen):
             return password
 
     def _handle_create_meeting(self, param):
-        self.save_progress('In action handler for: {0}'.format(self.get_action_identifier()))
+        self.save_progress("In action handler for: {0}".format(self.get_action_identifier()))
 
         action_result = self.add_action_result(ActionResult(dict(param)))
 
-        user_id = param['user_id']
-        password = self._get_password(param.get('password'), param.get('gen_password'))
-        waiting_room = param.get('waiting_room')
-        topic = param.get('topic')
-        agenda = param.get('agenda')
+        user_id = param["user_id"]
+        password = self._get_password(param.get("password"), param.get("gen_password"))
+        waiting_room = param.get("waiting_room")
+        topic = param.get("topic")
+        agenda = param.get("agenda")
+        alternative_hosts = param.get("alternative_hosts")
+        continuous_meeting_chat = param.get("continuous_meeting_chat", False)
+        auto_recording = param.get("auto_recording")
+        meeting_invitees = param.get("meeting_invitees")
 
         data = {}
 
-        input_param_dict = {
-            'password': password,
-            'topic': topic,
-            'agenda': agenda,
-        }
+        input_param_dict = {"password": password, "topic": topic, "agenda": agenda}
 
         for key, value in input_param_dict.items():
             if value:
                 data[key] = value
 
-        if waiting_room != 'None':
-            data['settings'] = {'waiting_room': (waiting_room == 'True')}
+        data["settings"] = {}
+
+        if waiting_room != "None":
+            data["settings"] = data["settings"] | {"waiting_room": (waiting_room == "True")}
```

Comment:
Why use this roundabout logic instead of just inserting the key?

```suggestion
            data["settings"]["waiting_room"] = waiting_room == "True"
```
Same below (where `.update` might be more appropriate

Reply:
done